resources/common.rb

```diff
@@ -0,0 +1,27 @@
+actions :install
+default_action :install
+
+# Creates the user if they do not already exist
+attribute :create_user, 'kind_of' => [TrueClass, FalseClass], :default => false
+
+# Create the owner, path, or group if they do not exist
+attribute :path, 'kind_of' => String, :default => '/opt/app/'
```

Comment:
Do we really want to define a default path location for applications? Seems like this should be nil by default and be required to be set. Same with the ``config_destination``. But maybe I'm wrong.